src/qml/qgismobileapp.qml

```diff
@@ -1159,6 +1169,33 @@ ApplicationWindow {
         gpsMenu.popup(locationToolbar.x + locationToolbar.width - gpsMenu.width, locationToolbar.y + locationToolbar.height - gpsMenu.height)
       }
 
+      function followLocation(forceRecenter) {
```

Comment:
I see some magic number 10 and 40. I think they can be converted to more descriptive constants. What I understood is:

```
FOLLOW_LOCATION_MIN_SCALE = 10
FOLLOW_LOCATION_MIN_SCREEN_MARGIN = 40
```